libtransmission/cache.cc

```diff
@@ -160,7 +160,7 @@ int Cache::writeBlock(tr_torrent_id_t tor_id, tr_block_index_t block, std::uniqu
     return cacheTrim();
 }
 
-Cache::CIter Cache::getBlock(tr_torrent const* torrent, tr_block_info::Location loc) noexcept
+Cache::CIter Cache::getBlock(tr_torrent const* torrent, tr_block_info::Location const& loc) noexcept
```

Comment:
I agree with the `const&`ing of Locations in this PR. `Location` is a uint64 + 4 uint32's